Fix focusSearchField(), fixes #374. Removes gear icon (and compose when too small), fixes #264, #313, #387, #402

hampustagerud opened the pull request.

hampustagerud committed:
Fixes #346

hampustagerud committed:
Merge remote-tracking branch 'rsms/master'

# Conflicts:
#	Messenger/AppDelegate.mm

hampustagerud committed:
Fixes focusSearchField()

him229 commented on Messenger/AppDelegate.mm:
```diff
@@ -124,7 +124,11 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
+=======
   DISABLE(webGLEnabled);
+>>>>>>> rsms/master
   
```
L127 - L131: I may be wrong, but did you mean to keep this there? (Referring to the merge conflict messages)

hampustagerud committed:
Update AppDelegate.mm

Missed conflicts in appdelegate

rsms commented:
This is great. Could you please explain how this fixes #346? I don't see the correlation.

him229 commented:
@rsms this is a fix for #374

hampustagerud commented:
Yes, this is a fix for #374. I don't know if it's still working, it's based on the assumption that the searchfield is the first element found by the query so it's in no way bulletproof.

hampustagerud committed:
Merge remote-tracking branch 'rsms/master'

hampustagerud committed:
Merge remote-tracking branch 'origin/master'

hampustagerud committed:
Disable overscroll in webview

This disables the overscroll of the UI which gives the app a very
non-native look

hampustagerud committed:
CSS based solution for hiding gear icon

Using a different selector and css I no longer experience any issues
with the gear icon being visible beneath the window controls

hampustagerud committed:
Fixes compose icon being shown beneath window controls

hampustagerud commented:
Added more commits, should now fix #264, #313, #387, #402. The gear is no longer showing at all when I use the app. I removed overscroll of the UI since it was giving the app a very non-native feel when you could scroll the entire frame.

I also included a working fix for #399 since there's not been an update yet.

dylankolson commented:
Any update on this?
